helm-go-lib/main.go

```diff
@@ -158,10 +164,16 @@ func installChart(namespace string, kubeToken string, apiServer string, releaseN
 }
 
 //export uninstallRelease
-func uninstallRelease(namespace, kubeToken, apiServer, releaseName string) *C.char {
-	settings := cli.New()
-	actionConfig := new(action.Configuration)
-	err := actionConfig.Init(settings.RESTClientGetter(), namespace, os.Getenv("HELM_DRIVER"), glog.Infof)
+func uninstallRelease(namespace string, kubeToken string, apiServer string, caFile string, releaseName string) *C.char {
+	var kubeConfig *genericclioptions.ConfigFlags
```

Comment:
would be nice to have this building config in a function

Reply:
Sure that makes sense